### DISCUSSION TITLE
在有牌子的直播间每日一条弹幕增加100亲密度

### DISCUSSION THREAD

**Discussion by nanako0922** [the1812/Bilibili-Evolved#2812] [功能建议 / Ideas]
在关注直播间每日第一条弹幕可以增加100牌子的亲密度，能否创建一键在所有关注且有牌子的直播间发送一条弹幕的功能
![image](https://user-images.githubusercontent.com/93066983/147842626-a7b44a6f-1411-45cc-b432-b6a4ace31cfa.png)

**Comment by Geeyun-JY3** [comment]
原来还有这么个机制

相关：https://github.com/the1812/Bilibili-Evolved/discussions/2717#discussioncomment-1810483

**Comment by Ethless** [comment]
well
